### ISSUE TITLE
Some stuff I need to tweak

### ISSUE THREAD

**Issue by nutterb** [nutterb/pixiedust#48]
### From having switched to checkmate

1. How do I want to handle partial matching or arguments?  I have it documented that `halign` does partial matching and will fail if multiple values are specified.  Having used `assertSubset`, I don't get that same behavior quite so easily.
2. I need a way to center, left align, or right align the entire table in HTML and LaTeX

### For bookdown support
1. Method to auto detect print method
2. table labels should reference chunk name, not pixie count

**Comment by nutterb** [comment]
I submitted this function to the `checkmate` repository.  It may or not be accepted there.  I should include it in `pixiedust` for now.  It at least gives me something to work with until something comes along that supports the `AssertCollection` object.

```r
assert_match_arg <- function(x, choices, several_ok = FALSE, ...)
{
  x <- choices[pmatch(x, choices, nomatch = 0L, duplicates.ok = several_ok)]
  if (several_ok) assertSubset(x, choices, ...)
  else assertChoice(x, choices, ...)
  x
}
```

**Comment by nutterb** [comment]
Implemented in 0.6.4
